microsoftgraph/MSGraph-SDK-Code-Generator
# releasing latest generation changes

- Bump NLog from 4.7.8 to 4.7.9
- Bump NLog.Config from 4.7.8 to 4.7.9
- Bump submodules/vipr from `d402360` to `adc288c`
- Bump submodules/vipr from `adc288c` to `882fcb0`
- Bump Microsoft.NET.Test.Sdk from 16.9.1 to 16.9.4
- Bump submodules/vipr from `882fcb0` to `f12f392`
- template script for generating launch.json
- add devcontainer to checkout recursively in codespaces
- recursive submodules for apidoctor->markdowndeep
- make output path one level up, fix excape character
- ordered object for deterministic output
- add ObjC
- output folder rename, test commands
- fix test data path
- update README
- fix typo
- add C# extension into container setup
- - fixes an issue where ODSP paths would not be encorded properly

@zengin I'm not sure what was the plan with this main branch. Are we planning to switch over from master in which case I should keep it updated or are we planning to stay on master as the trunk?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/519)
